subsys/mpsl/init/Kconfig

```diff
@@ -74,6 +74,29 @@ config MPSL_LOW_PRIO_IRQN
 	  This option sets the low priority interrupt that MPSL will use.
 	  Check the interrupt number in the MDK
 
+config MPSL_HFCLK_LATENCY
+	int
+	default 854 if SOC_SERIES_NRF54HX
+	default 854 if SOC_SERIES_NRF54LX
+	default 1400
+	help
+		This option configures the amount of time MPSL will assume it takes
```

Comment:
To be consistent with the the rest of the Kconfig options, the help text should be intended with two spaces from the initial tab indent